study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+DOM is not HTML, it looks like HTML because it is created directly from HTML.  DOM is also not the view source.  The view source just shows the HTML that makes up the page.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes, the information in DevTools are a visual respresentation of the DOM.
 ```
 
 In your own words, why do you think the DOM is important?
 
 ```md
-<!-- your answer here -->
+DOM is important because it is a model for structured document and it connects web pages to programming languages.
 ```
```

Comment:
The real power of the DOM lies in allowing us to interact with web pages. JavaScript provides us with many ways to interact with it!